pyk/src/pyk/prelude/kint.py

```diff
@@ -13,24 +13,323 @@
 
 
 def leInt(i1: KInner, i2: KInner) -> KApply:  # noqa: N802
+    r"""Instantiate the KAST term ```_<=Int_`(i1, i2)``.
+
+    Args:
+        i1: The left operand.
+        i2: The right operand.
+
+    Returns:
+        The KAST term ```_<=Int_`(i1, i2)``.
+    """
     return KApply('_<=Int_', i1, i2)
 
 
 def gtInt(i1: KInner, i2: KInner) -> KApply:  # noqa: N802
+    r"""Instantiate the KAST term ```_>Int_`(i1, i2)``.
+
+    Args:
+        i1: The left operand.
+        i2: The right operand.
+
+    Returns:
+        The KAST term ```_>Int_`(i1, i2)``.
+    """
     return KApply('_>Int_', i1, i2)
 
 
 def geInt(i1: KInner, i2: KInner) -> KApply:  # noqa: N802
+    r"""Instantiate the KAST term ```_>=Int_`(i1, i2)``.
+
+    Args:
+        i1: The left operand.
+        i2: The right operand.
+
+    Returns:
+        The KAST term ```_>=Int_`(i1, i2)``.
+    """
     return KApply('_>=Int_', i1, i2)
 
 
 def eqInt(i1: KInner, i2: KInner) -> KApply:  # noqa: N802
+    r"""Instantiate the KAST term ```_==Int_`(i1, i2)``.
+
+    Args:
+        i1: The left operand.
+        i2: The right operand.
+
+    Returns:
+        The KAST term ```_==Int_`(i1, i2)``.
+    """
     return KApply('_==Int_', i1, i2)
+
+
+def neqInt(i1: KInner, i2: KInner) -> KApply:  # noqa: N802
+    r"""Instantiate the KAST term ```_=/=Int_`(i1, i2)``.
+
+    Args:
+        i1: The left operand.
+        i2: The right operand.
+
+    Returns:
+        The KAST term ```_=/=Int_`(i1, i2)``.
+    """
+    return KApply('_=/=Int_', i1, i2)
+
+
+def notInt(i: KInner) -> KApply:  # noqa: N802
+    r"""Instantiate the KAST term ```~Int_`(i)``.
+
+    Args:
+        i: The integer operand.
+
+    Returns:
+        The KAST term ```Int_`(i)``.
+    """
+    return KApply('~Int_', i)
+
+
+def expInt(i1: KInner, i2: KInner) -> KApply:  # noqa: N802
+    r"""Instantiate the KAST term ```_^Int_`(i1, i2)``.
+
+    Args:
+        i1: The base operand.
+        i2: The exponent operand.
+
+    Returns:
+        The KAST term ```_^Int_`(i1, i2)``.
+    """
+    return KApply('_^Int_', i1, i2)
+
+
+def expModInt(i1: KInner, i2: KInner, i3: KInner) -> KApply:  # noqa: N802
+    r"""Instantiate the KAST term ```_^%Int__`(i1, i2, i3)``.
+
+    Args:
+        i1: The dividend operand.
+        i2: The divisior operand.
+        i3: The modulo divisor operand.
+
+    Returns:
+        The KAST term ```_^%Int__`(i1, i2, i3)``.
+    """
+    return KApply('_^%Int__', i1, i2, i3)
+
+
+def mulInt(i1: KInner, i2: KInner) -> KApply:  # noqa: N802
+    r"""Instantiate the KAST term ```_*Int_`(i1, i2)``.
+
+    Args:
+        i1: The left operand.
+        i2: The right operand.
+
+    Returns:
+        The KAST term ```_*Int_`(i1, i2)``.
+    """
+    return KApply('_*Int_', i1, i2)
+
+
+def divInt(i1: KInner, i2: KInner) -> KApply:  # noqa: N802
+    r"""Instantiate the KAST term ```_/Int_`(i1, i2)``.
+
+    Args:
+        i1: The dividend operand.
+        i2: The divisor operand.
```

Comment:
```suggestion
        i1: The dividend.
        i2: The divisor.
```

(Similarly for others.)